.github/pull_request_template.md

```diff
@@ -0,0 +1,22 @@
+# Welcome to Cryostat! Before contributing, make sure you have:
```

Comment:
Might need this to be h2 too?

Reply:
I put it at h1 to make the Welcoming to Cryostat the big heading of the pull request, but it's preference, I don't mind either way.

Reply:
Maybe splitting into 2 like

# Welcome to Cryostat!

## Before contributing, make sure you have: